site/src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copied from shadc/ui on 11/13/2024
+ * @see {@link https://ui.shadcn.com/docs/components/dialog}
+ */
+import * as DialogPrimitive from "@radix-ui/react-dialog";
+import { X } from "lucide-react";
```

Comment:
I think there are pros and cons either way. I like to easily tell what the icon visually looks like and the original name works well for that. Using icon in the name could also be a good way to look up all icons. Overall, I am not a fan of naming imported components for their specific purpose unless its really confusing otherwise.